src/test/test_utils.hpp

```diff
@@ -154,24 +154,24 @@ ovms::tensor_map_t prepareTensors(
     const std::unordered_map<std::string, ovms::Shape>&& tensors,
     ovms::Precision precision = ovms::Precision::FP32);
 
-void preparePredictRequest(tensorflow::serving::PredictRequest& request, inputs_info_t requestInputs, const std::vector<float>& data = {});
+void preparePredictRequest(tensorflow::serving::PredictRequest& request, inputs_info_t requestInputs, const std::vector<float>& data = {}, std::optional<int64_t> overrideBatch = std::nullopt);
```

Comment:
Wouldn't it be better to replace:
```test/test_utils.hpp:48:using inputs_info_t = std::map<std::string, std::tuple<ovms::shape_t, ovms::Precision>>;```
with
```test/test_utils.hpp:48:using inputs_info_t = std::map<std::string, std::tuple<std::vector<int64_t>, ovms::Precision>>;```

Reply:
Beside that LGTM

Reply:
done